server-spi/src/main/java/org/keycloak/models/utils/RoleUtils.java

```diff
@@ -40,17 +39,19 @@
      * @param targetGroup
      * @return true if targetGroup is in groups (directly or indirectly via parent child relationship)
      */
-    public static boolean isMember(Set<GroupModel> groups, GroupModel targetGroup) {
-        if (groups.contains(targetGroup)) return true;
+    public static boolean isMember(Stream<GroupModel> groups, GroupModel targetGroup) {
```

Comment:
Can we change method signatures in this file? It is in server-spi module, hence someone can use it. Should we provide a deprecated method with original signature?